examples/with_airflow/with_airflow/repository.py

```diff
@@ -25,17 +26,37 @@ def with_airflow():
 
 # end_repo_marker_0
 
-task_flow_repo = make_dagster_repo_from_airflow_dags_path(
-    os.path.abspath("./with_airflow/task_flow_dags/"),
-    "task_flow_repo",
+task_flow_dag_bag = DagBag(
+    dag_folder=os.path.abspath("./with_airflow/task_flow_dags/"),
+    include_examples=False,  # Exclude Airflow example dags
 )
+task_flow_schedules, task_flow_jobs = make_schedules_and_jobs_from_airflow_dag_bag(
+    task_flow_dag_bag,
+)
+
+
+@repository
+def task_flow_repo():
+    return [*task_flow_schedules, *task_flow_jobs]
+
 
+example_dag_bag = DagBag(
+    dag_folder="some/empty/folder/with/no/dags",
+    include_examples=True,  # Exclude Airflow example dags
+)
+example_schedules, example_jobs = make_schedules_and_jobs_from_airflow_dag_bag(
+    example_dag_bag,
+)
+
+
+@repository
+def airflow_examples_repo():
+    return [*example_schedules, *example_jobs]
 
```

Comment:
@odette-elementl I think you mentioned this. Are you going to move this to Definitions? I would wait until this lands to do that.